test/parallel/rolldown.test.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ *
+ * @group rolldown
```

Comment:
remove the group plz

Reply:
and move this test to the rollup folder!

Reply:
Done